Mathlib/RingTheory/MvPowerSeries/Evaluation.lean

```diff
@@ -60,6 +60,7 @@ variable {φ : R →+* S}
 open WithPiTopology
 
 /-- Families at which power series can be consistently evaluated -/
+@[mk_iff hasEval_def]
 structure HasEval (a : σ → S) : Prop where
```

Comment:
While we're at it, why not giving these two fields of `HasEval` a better name. At least, `hpow` could be `isTopologicallyNilpotent` or something approaching.